src/endpoints.ts

```diff
@@ -0,0 +1,25 @@
+import * as https from 'https';
```

Comment:
In this case i will:
- Resolve the comments here
- publish it as a beta rev
- Pull it into a branch on frontend  to check build sizes